src/components/trashlookup.jsx

```diff
@@ -47,6 +47,7 @@ class TrashLookUp extends Component {
 
     if (searchQuery.length > 0) {
       filtered = PostData.filter(m => m.address1.toLowerCase().indexOf(searchQuery.toLowerCase()) > -1);
+      console.log(filtered);
```

Comment:
please remove console.log, this should only be used for debugging.

Reply:
console.log is removed